src/components/ContentSwitcher/ContentSwitcher.story.jsx

```diff
@@ -1,3 +1,70 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, select } from '@storybook/addon-knobs';
+import { ContentSwitcher } from 'carbon-components-react';
+import List16 from '@carbon/icons-react/lib/list/16';
+import Code16 from '@carbon/icons-react/lib/code/16';
+import List20 from '@carbon/icons-react/lib/list/20';
+import Code20 from '@carbon/icons-react/lib/code/20';
+import List24 from '@carbon/icons-react/lib/list/24';
+import Code24 from '@carbon/icons-react/lib/code/24';
```

Comment:
Same here on the paths
```suggestion
import List16 from '@carbon/icons-react';
import Code16 from '@carbon/icons-react';
import List20 from '@carbon/icons-react';
import Code20 from '@carbon/icons-react';
import List24 from '@carbon/icons-react';
import Code24 from '@carbon/icons-react';
```